src/functionalTest/resources/responses/response-caseworker-submit-case-about-to-submit.json

```diff
@@ -66,6 +66,7 @@
       {
```

Comment:
why does your newly updated document with the comment not appear in the cicCaseApplicantDocumentsUploaded list in this response? Please can we validate the doc is added and the comment is present

Reply:
Added the document relevance to the response and the FT now passes - suspect the initial passing without the document relevance was to do with .when(IGNORING_EXTRA_FIELDS)